task02_iryna/src/dao/Paragraph.java

```diff
@@ -0,0 +1,36 @@
+package dao;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Paragraph implements CompositeWord{
+
+    public static int sentensesCounter;
```

Comment:
There is no good reason to declare a field "public" and "static" without also declaring it "final". Most of the time this is a kludge to share a state among several objects. But with this approach, any object can do whatever it wants with the shared state, such as setting it to null.